### PR TITLE
Fix Tutorial 2 for GTK+

### PR DESCRIPTION
**Pull request by danyeaw**
Fixes two Attribute Errors I was getting when trying to run tutorial 2. May also fix issue #71.

### CODE REVIEW

File: src/gtk/toga_gtk/widgets/base.py [hunk 1]
```diff
@@ -40,11 +40,11 @@ def container(self, container):
 
     @property
     def enabled(self):
-        return Gtk.gtk_widget_get_sensitive(self.native)
+        return Gtk.Widget.get_sensitive(self.native)
```
Comment (freakboy3742) [inline src/gtk/toga_gtk/widgets/base.py]: Shouldn't this be `self.native.get_sensitive()`?

File: src/gtk/toga_gtk/widgets/base.py [hunk 2]
```diff
@@ -40,11 +40,11 @@ def container(self, container):
...
 
     @enabled.setter
     def enabled(self, value):
-        Gtk.gtk_widget_set_sensitive(self.native, value)
+        Gtk.Widget.set_sensitive(self.native, value)
```
Comment (freakboy3742) [inline src/gtk/toga_gtk/widgets/base.py]: Similarly, shouldn't this be `self.native.set_sensitive(value)`?

File: src/core/toga/widgets/icon.py [hunk 3]
```diff
@@ -39,7 +39,7 @@ def __init__(self, path, system=False, factory=None):
         else:
             self.filename = self.path
 
-        self._impl = self.factory.Icon(interface=self)
+        self._impl = self.factory.Icon(interface=self, filename=self.filename)
```
Comment (freakboy3742) [inline src/core/toga/widgets/icon.py]: This doesn't seem right: you should be able to get to the filename by accessing the interface, without having to pass it in explicitly...

**Comment by danyeaw** [comment]
Thanks @freakboy3742! Changes implemented. I learned a few things from your feedback, so thanks for your patience.

**Pull request merged**